Exiled.Events/EventArgs/RoundEndedEventArgs.cs

```diff
@@ -43,5 +44,13 @@ public RoundEndedEventArgs(LeadingTeam leadingTeam, RoundSummary.SumInfo_ClassLi
         /// Gets or sets the time to restart the next round.
         /// </summary>
         public int TimeToRestart { get; set; }
+
+        /// <summary>
+        /// Resync's player information in Player class.
+        /// </summary>
+        public void ReSyncPlayerInformation()
```

Comment:
Why?

Reply:
Oh, for modded

Reply:
![image](https://user-images.githubusercontent.com/24619207/177050703-0d62528f-84bc-41fc-889c-39bbc2a8c36b.png)

Reply:
Technically, it's a wrapper just for people to not get confused, but they can also just do Player.ReSyncAllInstantiatedContainers(); so I can remove it but I offered it so I did it

Reply:
As for me it makes no sense, the round will exist for some time after the call of this event, and the players still have roles(logically)

Reply:
I mean, I agree. I made that point earlier that the data will be reset anyway on server restart - If you want me to remove it I can, I only added it as a helpful call per Modded's request